requirements/big_mode.txt

```diff
@@ -1,5 +1,5 @@
 # Requirements for Redis and PostgreSQL support
-channels-redis>=2.2,<2.4
+channels-redis>=2.2,<2.5
```

Comment:
channels-redis uses pools now:
https://github.com/django/channels_redis/blob/8efee75564ef36dd5388cfbcaff5bc5b12e76a0b/CHANGELOG.txt#L35

I don't think there will be problems with our redis pool. But maybe we could refactor our pool to use the channels pool:
https://github.com/OpenSlides/OpenSlides/blob/bb41125bd146af1e367fa0d9eeeb3adf49646cea/openslides/utils/redis.py#L36:L54

Reply:
We are currently using these pools: https://github.com/OpenSlides/OpenSlides/blob/bb41125bd146af1e367fa0d9eeeb3adf49646cea/openslides/utils/redis_connection_pool.py#L5